src/fields/clmul_1byte.cpp

```diff
@@ -6,7 +6,7 @@
 
 /* This file was substantially auto-generated by doc/gen_params.sage. */
 
-#ifdef __x86_64
+#ifdef HAVE_CLMUL
```

Comment:
Now that these files are conditionally built, is there any reason for these guards?

Reply:
I guess not.